GitBranches/Program.cs

```diff
@@ -12,8 +12,8 @@ static void Main(string[] args)
         {
             Console.WriteLine("Demo Git");
             Console.WriteLine("Estas son las preguntas que vamos a contestar: ");
-            Console.WriteLine("1. Cunato timepo llevas en Hartmann?");
-            Console.WriteLine("Esta es la rama de Robert");
```

Comment:
The idea is to have both answers. Do not erase Robert answers please

Reply:
Fixed